index.rst

```diff
@@ -520,14 +534,36 @@ Naturally, people should be able to fork and customize the app for specific impl
 
 Functionality of the tool could include:
 
-- Ability to flip through a 2-d data cube as a movie
+- Ability to flip through a 2-dimensional data cube as a movie
 - Click on a given amp and have a plot of the value versus time, with the expectation value of the metric over plotted etc.
 - Ability to show sky maps as a function of time, and adjust the binning on-the-fly
-- Capable of mining the appropriate data given the specific analysis_tools metric (including timestamp etc)
+- Capable of mining the appropriate data given the specific analysis_tools metric (including timestamp, etc.)
 
 Lastly, it is recognized that the DM DRP team also needs to interact with non-aggregated data, this is outside the scope of FAFF, however, adopting a common toolset, or one that is based off the tooling being discussed here is recommended.
 
 
+On-the-fly Quality Assessment and Alerts
```

Comment:
I think this subsection would fit better as part of the preceding section "Deliverable 3: Interacting with Rapid Analysis Data and Metrics"

Reply:
Ya, I went back and forth on this... 
My reason for including this here was that it mentions "present it to the stakeholders" which is sort of an alert.
Moved back.